apiary.apib

```diff
@@ -1010,6 +1010,10 @@ All buckets are returned.
                     },
                     "updated": "2024-11-28T13:03:13+0100",
                     "color": null
+                    "backendPath": [
```

Comment:
Ty responses pro bucket sů různě outdated, zrovna sem založil risk https://keboola.atlassian.net/browse/RISK-64 je to fakt na hovno